src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+[Bing](https://www.bing.com/toolbox/webmaster)'s or
+[Yandex](https://webmaster.yandex.com/welcome/)'s webmaster tools, and
+consider including [structured data](https://goo.gle/search-gallery) via
+schemas from [Schema.org](https://schema.org/) in your PWA.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Works with any input type
+
+Your PWA is equally usable with a mouse as it is with a keyboard, by touch,
+or with a stylus.
+
+{% endDetailsSummary %}
+
+Your PWA is equally usable with a mouse as it is with a keyboard, by touch,
+or with a stylus.
+
+#### Why
+
+Devices offer a variety of input methods, and users should be able to
+seamlessly switch between them while using your application. Just as important,
+input methods should not depend on screen size, meaning that large viewports
+need to support touch and small viewports need to support keyboards and mice.
+To the best of your ability, ensure that your application and all of its
+features supports usage of any input method your user may choose to use.
+Where appropriate, you should also enhance experiences to allow for
+input-specific controls as well (such as pull-to-refresh).
+
+#### How
+
+The [Pointer Events API](https://developers.google.com/web/updates/2016/10/pointer-events)
+provides a unified interface for working with various input options, and is
+especially good for adding stylus support. For supporting both touch and
+keyboard, ensure that you're using the correct semantic elements (anchors,
+buttons, form controls, etc.) and not rebuilding these with non-semantic
+HTML (which is good for accessibility). When including interactions that
+activate on hover, ensure they can activate on click or tap, too.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Provides context for permission requests
+
+When asking permission to use powerful APIs, provide context and ask only
+when the API is needed.
+
+{% endDetailsSummary %}
+
+When asking permission to use powerful APIs, provide context and ask only
+when the API is needed.
+
+#### Why
+
+APIs that trigger a permission prompt, like notifications, geolocation, and
+credentials, are intentionally designed to be disruptive to a user as they
+tend to be related to powerful functionality that requires opt-in. Triggering
+these prompts without additional context, like on page load, makes users
+less likely to accept those permissions and more likely to distrust them
+in the future. Instead, only trigger those prompts after providing an
+in-context rationale to the user for why you need that permission.
+
+#### How
+
+The [Permission UX](https://developers.google.com/web/fundamentals/push-notifications/permission-ux)
+article and UX Planet's
+[The Right Ways to Ask Users for Permissions](https://uxplanet.org/mobile-ux-design-the-right-ways-to-ask-users-for-permissions-6cdd9ab25c27)
+are good resources to understand how to design permission prompts that, while
```

Comment:
'are good resources to understand how to design'

Change to:

'describe designing'

Reply:
Going to leave as if unless major objections.